Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update
 
 # Install pip and poetry
 RUN pip install --upgrade pip
-RUN pip install "poetry==1.2.2"
+RUN pip install "poetry==1.8.2"
```

Comment:
There is a later version now I've got 1.8.3 locally.

Reply:
✅